pontoon/base/models.py

```diff
@@ -1541,15 +1541,18 @@ def calculate_stats(self, save=True):
         fuzzy = translations.filter(fuzzy=True).count()
 
         # Plural
+        half_translated = 0
         nplurals = locale.nplurals or 1
         for e in translated_entities.exclude(string_plural=''):
             translations = Translation.objects.filter(entity=e, locale=locale)
             if translations.filter(approved=True).count() == nplurals:
                 approved += 1
             elif translations.filter(fuzzy=True).count() == nplurals:
                 fuzzy += 1
+            else:
+                half_translated += 1
```

Comment:
This `else` block will also catch strings that have suggestions for all plural forms, in which case we need to count them as suggested instead of missing. So they should not be counted as `partially_translated`.

To properly count `partially_translated` strings, you need to make sure that at least one plural form doesn't have any translations.